scipy/misc/__init__.py

```diff
@@ -32,13 +28,25 @@
    imshow - Simple showing of an image through an external viewer [requires Pillow]
    info - Get help information for a function, class, or module
    lena - Get classic image processing example image Lena
+   toimage - Takes a numpy array and returns a PIL image [requires Pillow]
+   source - Print function source code
```

Comment:
Agreed, `np.info` feels somehow in the same league as matplotlib's pylab: was possibly useful, not so much anymore. 

So I added deprecation warnings to all of `info`, `source` and `who`.